media_driver/agnostic/common/codec/hal/codechal_encode_scalability.cpp

```diff
@@ -121,6 +121,8 @@ MOS_STATUS CodecHalEncodeScalability_PopulateHintParams(
     CODECHAL_ENCODE_CHK_NULL_RETURN(cmdBuffer);
     PMOS_CMD_BUF_ATTRI_VE pAttriVe = (PMOS_CMD_BUF_ATTRI_VE)(cmdBuffer->Attributes.pAttriVe);
 
+    if (pAttriVe)
```

Comment:
Again, a patch we should consider merge in right now.

Also, you have 2 patches with the same name. That's bad. Either squash them or add something to separate titles, for example, add "decode" and "encode" prefix:
`encode: fix NULL pointer access issue for encode virtual engine enabling`

Reply:
ok.

Reply:
Done with PR   https://github.com/intel/media-driver/pull/481